Modify S2078(multiple languages): Update to the education framework (APPSEC-115) (#1209)
SonarSource/rspec: rules/S2078/common/fix/validation.adoc

==== Validation

As a rule of thumb, the best approach to protect against injections is to
systematically ensure that untrusted data cannot break out of the initially
intended logic.

For LDAP injections, the cleanest way to do so is to use parameterized queries

> if it is available for your use case. The second cleanest way to do so is to

Reviewer: ```suggestion
if it is available for your use case. An alternative is to
```
